content/chainguard/chainguard-images/working-with-images/security-advisories/how-chainguard-issues/index.md

```diff
@@ -0,0 +1,78 @@
+
+![NVD notice stating a vulnerability has not yet been analyzed](nvd-analyze.png)
+
+In addition to the NVD, other vulnerability databases, such as the [GitHub Advisory Database](https://github.com/advisories) (GHSA) and the [Go Vulnerability Database](https://vuln.go.dev/) are also referenced by vulnerability scanners.
+
+
+### Step 3: Advisory is Issued
+
+Once a CVE has been reviewed by the NVD, it will be picked up by vulnerability scanners and reported in any affected container images. Chainguard uses [Grype](https://github.com/anchore/grype), an open-source vulnerability scanner from Anchore, as its primary tool for vulnerability detection.
+
+The newly detected CVE is then moved into the next phase where it waits for a team member to assess it. A security advisory will be issued with the status of “Under Investigation” to alert downstream users that Chainguard is aware of its presence. Security advisories are issued per package, as one CVE may impact different packages in different ways. This security advisory will be updated over time, as shown in the next step.
+
+
+### Step 4: Advisory is Updated
+
+With an advisory issued for the package, further investigation is often needed to determine the impact of the CVE. In some cases, it will be determined that the CVE is not truly present in the package, therefore making it a [false positive vulnerability](/chainguard/chainguard-images/recommended-practices/false-results/). The associated security advisory would have its status updated to “Not Affected”, and further updates to the advisory would not occur.
+
+If the vulnerability is a true positive finding, then it is present in the package and further action must be taken. When an  upstream fix is available, such as a newer package version which remediates the CVE, then this update will be made, and the advisory modified to state the vulnerability is now “Fixed”. Chainguard may even proactively bump a vulnerable dependency to its newer, patched version before upstream projects have done so themselves. Or, patches issued outside of the upstream repository may be applied to remediate the vulnerability when a new package version is not yet available. Note that an end user would need to pull the new version of the container image with the updated fix for the CVE as older versions of the image would still be vulnerable. 
+
+In some cases, a fix for the CVE may not yet be available. A package will be marked as having the “Pending upstream fix” status in this situation. Once an upstream fix is released, then it will be applied to the package, and the advisory status updated to “Fixed”.
```

Comment:
```suggestion
In some cases, a fix for the CVE may not yet be available. A package will be marked as having the "Pending upstream fix" status in this situation. Once an upstream fix is released, then it will be applied to the package and the advisory status updated to "Fixed".
```